packages/error-reporting/utils/errors-api-transport.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var AuthClient = require('../src/google-apis/auth-client.js');
+
+/* @const {String} Base Error Reporting API */
+var API = 'https://clouderrorreporting.googleapis.com/v1beta1/projects';
+
+var ONE_HOUR_API = 'timeRange.period=PERIOD_1_HOUR';
+
+class ErrorsApiTransport extends AuthClient {
+  constructor(config, logger) {
+    super(config, logger);
+  }
+
+  deleteAllEvents(cb) {
+    var self = this;
+    self.getProjectId(function(err, id) {
+      if (err) {
+        return cb(err);
+      }
+
+      var options = {
+        uri: [API, id, 'events'].join('/'),
+        method: 'DELETE'
+      };
+      self.request_(options, function(err, /* jshint unused:false */ response,
```

Comment:
>  I thought the packages just directly used the .jshhintrc from the root of the repo.

That is the case, but I think error-reporting is too large of a code base where the effort to conform to the parent style conventions is not worth it. The point of being consistent between them is to make maintenance and contribution easier, but:

  1. we're not doing a lot of cross-maintenance (GCN devs in error-reporting, or error-reporting in the other areas of GCN), and
  2. the potential outside contributors for error-reporting aren't necessarily direct users of the other libraries in this repo

As long as our CI can run the necessary commands (doc generation, linting, tests: unit, snippet, and system) in E-R, I think it's safe for you to operate under your own definition of best practices and conventions.

The parent `npm run lint` task *might* need to be re-engineered to accept a custom .jshintrc file for E-R, but I don't know. Feel free to make any necessary changes.